pkg/ddevapp/ddevapp.go

```diff
@@ -2061,6 +2061,12 @@ func (app *DdevApp) DockerEnv() {
 	// * provide default host-side port bindings, assuming only one project running,
 	//   as is usual on Gitpod, but if more than one project, can override with normal
 	//   config.yaml settings.
+	// Codespaces stumbles if not on a "standard" port like port 80
+	if nodeps.IsGitpod() || nodeps.IsCodespaces() {
```

Comment:
What I intended:

```suggestion
	if nodeps.IsCodespaces() {
```

Thanks as always for looking closely.